src/main.rs

```diff
@@ -194,6 +240,9 @@ pub struct Alert {
 
     #[serde(rename = "endsAt")]
     ends_at: String,
+
+    #[serde(rename = "generatorURL")]
```

Comment:
What's this for? It's the only occurrance of generatorURL in the code I can see?

Reply:
I've added some more keys from [the webhook example from the documentation](https://prometheus.io/docs/alerting/configuration/). This is one of them.